code/backend/batch/BatchStartProcessing.py

```diff
@@ -26,11 +27,13 @@ def batch_start_processing(req: func.HttpRequest) -> func.HttpResponse:
     )
     files_data = list(map(lambda x: {"filename": x["filename"]}, files_data))
     # Create the QueueClient object
-    queue_client = QueueClient.from_connection_string(
-        azure_blob_storage_client.connect_str,
-        env_helper.DOCUMENT_PROCESSING_QUEUE_NAME,
+    queue_client = QueueClient(
+        account_url=f"https://{env_helper.AZURE_BLOB_ACCOUNT_NAME}.queue.core.windows.net/",
+        queue_name=env_helper.DOCUMENT_PROCESSING_QUEUE_NAME,
+        credential=DefaultAzureCredential(),
```

Comment:
Does this still work when not using RBAC?

Reply:
Yep, I've tested that

Reply:
(I've mentioned this in the PR description but dunno if this is the place people typically look)

Reply:
Oh no actually I've realised that this might not be correct, because I misunderstood how the solution is deployed when RBAC can't be used. I will rework this